examples/multi-destination.ts

```diff
@@ -59,21 +98,21 @@ const main = async ({
 	await new Promise((resolve) => {
 		deviceScanner.on('ready', resolve);
 	});
-	const source: sourceDestination.SourceDestination = new sourceDestination.File(
-		{
-			path: sourceImage,
-		},
-	);
 	let configure: ConfigureFunction | undefined;
 	if (config !== undefined) {
 		configure = async (disk: Disk) => {
 			await legacyConfigure(disk, { config: await readJsonFile(config) });
 		};
 	}
-	const destinationDrives = Array.from(deviceScanner.drives.values()).filter(
-		(drive) => {
-			return devices.includes(drive.device!);
-		},
+	const destinationDrives = await Promise.all(
+		devices.map((d) => openUrl(d, deviceScanner, true)),
+	);
+	const source = await openUrl(
+		sourceImage,
+		deviceScanner,
+		false,
+		// ConfiguredSource will not work with O_DIRECT
+		!(trim || configure !== undefined),
```

Comment:
I think it's a bit clearer as
```suggestion
		!trim && configure === undefined,
```